includes/admin/class-wc-rest-payments-settings-controller.php

```diff
@@ -101,10 +121,34 @@ public function get_settings(): WP_REST_Response {
 	public function update_settings( WP_REST_Request $request ) {
 		$this->update_is_wcpay_enabled( $request );
 		$this->update_enabled_payment_methods( $request );
+		$this->update_is_digital_wallets_enabled( $request );
+		$this->update_digital_wallets_enabled_locations( $request );
 
 		return new WP_REST_Response( [], 200 );
 	}
 
+	/**
+	 * Validate the digital wallets enabled locations are supported and their values
+	 * are boolean.
+	 *
+	 * @param object          $value Value to check.
+	 * @param WP_REST_Request $request The request sent to the WP REST API.
+	 * @param string          $param Name of the parameter passed to endpoint holding $value.
+	 * @return bool
+	 */
+	public static function validate_digital_wallets_enabled_locations( $value, $request, $param ) {
```

Comment:
Since the other arguments are not used.

```suggestion
	 * @param object $value Value to check.
	 * 
	 * @return bool
	 */
	public static function validate_digital_wallets_enabled_locations( $value ) {
```